flaxandteal/coral-arches
# Fix (#2069): Consultation assign to with dashboard functionality

### Description
Update the assigned by to assigned to. Uses the 'assignee' 'assigned to' node so that the assigned users are visible in both the planning consultation and the planning consultation response

Change in node group has been applied to the dashboard to assign the tasks correctly

Dashboard has been refactored to use the strategy pattern

Workflow slugs have been updated to allow the correct workflow to open for the correct group (consultation or response)

### Tests
- unregisted Notify Planning Function and re register it
- reload Consultation
- coral reload
- Check Assign to is visible on the planning consultation and updates the action nodegroup
- Check Assign to is visible on the response workflows and updates the action nodegroup
- Run through the dashboard test sheets for dev to ensure dashboard tasks are showing
- Run through planning consultation test for the assigned to button on the actions tab
- Run through Dashboard Notifications
